src/neo/SmartContract/ApplicationEngine.cs

```diff
@@ -333,11 +336,11 @@ private static Block CreateDummyBlock(StoreView snapshot)
             };
         }
 
-        private static InteropDescriptor Register(string name, string handler, long fixedPrice, CallFlags requiredCallFlags, bool allowCallback)
+        private static InteropDescriptor Register(string name, string handler, long fixedCPUPrice, long fixedStoragePrice, CallFlags requiredCallFlags, bool allowCallback)
```

Comment:
Is `fixedStoragePrice` always `0`?

Reply:
See `System.Storage.Delete`. I think we can use more of it, probably for `Storage` related things and for `Blockchain` getters.

Reply:
Maybe we should remove `fixedStoragePrice` and use `AddGas` in `System.Storage.Delete`.

Reply:
Well, if we're not using it for other syscalls then sure yes, it's easier to change `System.Storage.Delete` and have one price field.

Reply:
Let's do it.

Reply:
Updated.